mempool/clist_mempool_test.go

```diff
@@ -97,12 +97,13 @@ func checkPriorityTxs(t *testing.T, mempool Mempool, count int, peerID uint16) t
 	txInfo := TxInfo{SenderID: peerID}
 	for i := 0; i < count; i++ {
 		txBytes := make([]byte, 20)
-		txBytes = tx_extensions.PrependBytes(txBytes) // Convert to priority tx (dkg)
-		txs[i] = txBytes
 		_, err := rand.Read(txBytes)
 		if err != nil {
 			t.Error(err)
 		}
+		txBytes = tx_extensions.PrependBytes(txBytes) // Convert to priority tx (dkg)
+		assert.True(t, isPriority(txBytes))
+		txs[i] = txBytes
```

Comment:
Having rand after the prepend was messing up prefix and made the tx no longer a priority tx

Reply:
👍